topics/metagenomics/tutorials/beer-data-analysis/tutorial.md

```diff
@@ -0,0 +1,391 @@
+
+> ### {% icon details %} Kraken2
+>
+> This tool uses the minimizer method to sample the k-mers (all the read's subsequences of length $$k$$) in a deterministic fashion in order to reduce memory constumption and processing time. In addition, it masks low-complexity sequences from reference sequences by using dustmasker.
+>
+> ![Kraken2](./images/kraken2.jpg)
+>
+{: .details}
+
+> ### {% icon hands_on %} Hands-on: Kraken2
+>
+> 1. {% tool [Kraken2](toolshed.g2.bx.psu.edu/repos/iuc/kraken2/kraken2/2.0.8_beta+galaxy0) %} with the following parameters:
+>    - *"Single or paired reads"*: `Single`
+>        - {% icon param-file %} *"Input sequences"*: Output of **fastp**
+>    - *"Print scientific names instead of just taxids"*: `Yes`
+>    - In *"Create Report"*:
+>        - *"Print a report with aggregrate counts/clade to file"*: `Yes`
+>        - *"Format report output like Kraken 1's kraken-mpa-report"*: `Yes`
+>
+{: .hands_on}
+
+# Visualize the community
+
+Once we have assigned the corresponding taxa to the sequences, the next step is to properly visualize the data: visualize the diversity of taxons at different levels.
+
+To do that, we will use the tool **Krona** ({% cite Ondov_2011 %}). But before that, we need to adjust the format of the data output from Kraken2.
+
+> ### {% icon hands_on %} Hands-on: Prepare dataset for Krona
+>
+> 1. {% tool [Reverse](toolshed.g2.bx.psu.edu/repos/iuc/datamash_reverse/datamash_reverse/1.1.0) %} with the following parameters:
+>    - {% icon param-file %} *"Input tabular dataset"*: output of **Kraken2** {% icon tool %})
+> 2. {% tool [Replace Text](toolshed.g2.bx.psu.edu/repos/bgruening/text_processing/tp_replace_in_line/1.1.2) %} with the following parameters:
+>    - {% icon param-file %} *"File to process"*: `out_file` (output of **Reverse** {% icon tool %})
+>    - In *"Replacement"*:
+>        - {% icon param-repeat %} *"Insert Replacement"*
+>            - *"Find pattern"*: `\|`
+>            - *"Replace with:"*: `\t`
+{: .hands_on}
+
+**Krona** creates an interactive report that allows hierarchical data (like taxonomy) to be explored with zooming, multi-layered pie charts. With this tool, we can easily visualize the composition of the microbiome communities and also compare how the populations of microorganisms are modified between different samples.
+
+> ### {% icon hands_on %} Hands-on: Krona pie chart
+>
+> 1. {% tool [Krona pie chart](toolshed.g2.bx.psu.edu/repos/crs4/taxonomy_krona_chart/taxonomy_krona_chart/2.7.1) %} with the following parameters:
+>    - *"What is the type of your input data"*: `Tabular`
+>        - {% icon param-file %} *"Input file"*:  output of **Replace Text** {% icon tool %}
+>
+> 2. Inspect the generated file
+{: .hands_on}
+
+Let's take a look at the result.
+
+<!--
+<iframe id="krona" src="krona_all.html" frameBorder="0" width="100%" height="900px"> ![Krona](./images/krona.png) </iframe>
```

Comment:
if you include the krona html file here instead of a png it would be interactive and embedded, could be nice?